File: snippets/js/array/merge add/merge.js
var array1 = [3, 2, 1];
var array2 = [100, 200, 300];

var array = array1.concat(array2);   // ≡
console.log(array);  // [ 3, 2, 1, 100, 200, 300 ]

// expand:
var array = [...array1, ...array2];  // ≡
console.log(array);  // [ 3, 2, 1, 100, 200, 300 ]

File: snippets/py/array/merge add/merge.py
array1 = [3, 2, 1]
array2 = [100, 200, 300]
array1.extend(array2)    # ≡
print(array1)  # [ 3, 2, 1, 100, 200, 300 ]

# add:
array1 = [3, 2, 1]
array2 = [100, 200, 300]
array = array1 + array2  # ≡
print(array)  # [ 3, 2, 1, 100, 200, 300 ]